### PR TITLE
Docs and biopax

### PR DESCRIPTION
**Pull request by bgyori**
First of all, the initial implementation of the BioPAX API and processor.
Second, sphinx module documentation set up.
Third, many small changes to other parts of the project.

### CODE REVIEW

File: belpy/biopax/processor.py [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+        stmts = self._get_generic_modification('acetyl')
+        for s in stmts:
+            self.belpy_stmts.append(Acetylation(*s))
+
+    def get_glycosylation(self):
+        stmts = self._get_generic_modification('glycosyl')
+        for s in stmts:
+            self.belpy_stmts.append(Glycosylation(*s))
+
+    def get_palmiotylation(self):
```
Comment (johnbachman) [inline belpy/biopax/processor.py]: Spelling error--should be palmitoylation here and on line 100.

Comment (bgyori) [inline belpy/biopax/processor.py]: Okay, I'll fix that.

**Comment by bgyori** [comment]
Should I put in anymore changes or can we merge?

**Comment by johnbachman** [comment]
Looks good.

**Pull request merged**